target/linux/octeon/image/Makefile

```diff
@@ -47,6 +51,19 @@ define Device/ubnt_edgerouter
 endef
 TARGET_DEVICES += ubnt_edgerouter
 
+define Device/ubnt_edgerouter-4
+  DEVICE_VENDOR := Ubiquiti
+  DEVICE_MODEL := EdgeRouter 4
+  SOC := cn7130
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_DTS := $$(SOC)_$(1)
```

Comment:
If defined explicitly anyway, I'd prefer just the string here, as there is no benefit in using the variables anymore.

Reply:
sure, dropped `SOC` variable since it is not used anywhere outside `DEVICE_DTS` and defined `DEVICE_DTS` statically. also reused `DEVICE_DTS` in `KERNEL_DEPENDS` and `append-dtb-to-elf` function